assets/base/i18n/en.json

```diff
@@ -21,6 +21,8 @@
   "about.title": "About Mattermost",
   "about.version": "Mattermost Version:",
   "access_history.title": "Access History",
+  "action_menu.select": "Select an option",
```

Comment:
These should have a `mobile.` prefix or add them in the webapp

Reply:
Cool, will add